### PR TITLE
indestructible windows cant be destroyed by ingame means anymore

### LINKED ISSUE #72271: RCDs can remove indestructible windows
## Reproduction:
1. Place down an `/obj/structure/window/reinforced/fulltile/indestructible`
2. RCD it away
3. It's gone

### PR DESCRIPTION
**Pull request by Salex08**
## About The Pull Request

closes: https://github.com/tgstation/tgstation/issues/72271

I tested on a local server to see if there are more indestructible things the rcd can destroy couldnt find any further.
Also makes sure indestructible windows cant be destroyed by any unintentional means

## Why It's Good For The Game

bug fix

## Changelog

:cl:
fix: indestructible windows cant be destroyed by ingame means anymore
/:cl:

### CODE REVIEW

**Comment by Fikou** [comment]
can ya fix it for fire axes too?

**Comment by Salex08** [comment]
> can ya fix it for fire axes too?

you mean firelocks?

**Comment by Fikou** [comment]
> > can ya fix it for fire axes too?
> 
> you mean firelocks?

no, fire axes can break indestructible windows

**Comment by Salex08** [comment]
> > > can ya fix it for fire axes too?
> > 
> > 
> > you mean firelocks?
> 
> no, fire axes can break indestructible windows

ahhh sorry, thats weird I will be on it

**Comment by SyncIt21** [comment]
I KNEW IT!!

A lot of other PR's have been failing for the exact same reason

Screenshot (70)

I have a strong feeling it was caused by #72033

**Comment by Salex08** [comment]
> I KNEW IT!!
> 
> A lot of other PR's have been failing for the exact same reason
> 
> Screenshot (70)
> 
> I have a hunch it was caused by #72033 but it also doesn't make sense cause all tests were passing yesterday

well am i supposed to empty commit until it eventually stops giving me unrelated error?

**Comment by Salex08** [comment]
> > > can ya fix it for fire axes too?
> > 
> > 
> > you mean firelocks?
> 
> no, fire axes can break indestructible windows

ok done

**Comment by SyncIt21** [comment]
> > I KNEW IT!!
> > A lot of other PR's have been failing for the exact same reason
> > Screenshot (70)
> > I have a hunch it was caused by #72033 but it also doesn't make sense cause all tests were passing yesterday
> 
> well am i supposed to empty commit until it eventually stops giving me unrelated error?

Did that didn't work.  Have to wait until the guy whie messed up magic cards or something else gets here.
U have nothing to worry about

**Comment by Mothblocks** [comment]
its not you ill deal with it latre

**Comment by Jacquerel** [comment]
> its not you ill deal with it latre

I think #72266 fixes it

File: code/game/objects/items/fireaxe.dm [hunk 1]
```diff
@@ -57,8 +57,9 @@
 		return
 	if(HAS_TRAIT(src, TRAIT_WIELDED)) //destroys windows and grilles in one hit
 		if(istype(A, /obj/structure/window) || istype(A, /obj/structure/grille))
-			var/obj/structure/W = A
-			W.atom_destruction("fireaxe")
+			if(!A.resistance_flags && INDESTRUCTIBLE)
```
Comment (ZephyrTFA) [inline code/game/objects/items/fireaxe.dm:60]:
```suggestion
			if(!(A.resistance_flags & INDESTRUCTIBLE))
```

File: code/modules/awaymissions/mission_code/murderdome.dm [hunk 2]
```diff
@@ -4,10 +4,17 @@
 	flags_1 = PREVENT_CLICK_UNDER_1 | NODECONSTRUCT_1
 	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
 
+/obj/structure/window/reinforced/fulltile/indestructible/rcd_vals(mob/user, obj/item/construction/rcd/the_rcd)
+	return FALSE
+
+
```
Comment (ZephyrTFA) [inline code/modules/awaymissions/mission_code/murderdome.dm:10]:
```suggestion
	return FALSE
```

**Review by ZephyrTFA — CHANGES_REQUESTED** [review]

**Comment by lessthnthree** [comment]
So if the tram currently has
resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF

Is that going to make it actually indestructible now? That's not ideal and would have to change.

**Comment by Salex08** [comment]
> So if the tram currently has resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
> 
> Is that going to make it actually indestructible now? That's not ideal and would have to change.

no it doesnt look closely at code

`if(istype(A, /obj/structure/window) || istype(A, /obj/structure/grille))`

**Comment by ZephyrTFA** [comment]
> So if the tram currently has resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
> 
> Is that going to make it actually indestructible now? That's not ideal and would have to change.

I'm confused as to where you've supposedly seen this

**Review by ZephyrTFA — APPROVED** [review]

**Pull request merged**